src/app/coursegrab/controllers/update_device_token_controller.py

```diff
@@ -3,7 +3,7 @@
 
 class UpdateDeviceTokenController(AppDevController):
     def get_path(self):
-        return "/users/device-token/"
+        return "/users/update-device-token/"
```

Comment:
I think we should keep the urls `/users/device-token/` and `/users/notification/`. But everything else looks good!